sample-compose/app/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,6 @@
     package="co.nimblehq.sample.compose">
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
Should we request for both`CAMERA` & `LOCATION` permissions, so it's the same with `sample-xml`? 🙏

Reply:
@Tuubz As this is a sample, yessss

Reply:
@Tuubz I have re-thought this issue, according to this documentation https://google.github.io/accompanist/permissions/#rememberpermissionstate-and-remembermultiplepermissionsstate-apis, we can easily do `multiple permission request` by using `rememberMultiplePermissionsState`. However, applying `rememberMultiplePermissionsState` requires replacing the current sample with `single permission request`. So I think we just need to make a sample for one of them. I don't think we have enough effort to make many samples for all cases and maintain them later 🙏 

IMO, I will ignore this man.

Reply:
> However, applying rememberMultiplePermissionsState requires replacing the current sample with single permission request

Could you please explain a bit more on this? 🙇  

I thought we just had to simply change to:

`val cameraPermissionState = rememberMultiplePermissionsState(listOf(CAMERA, LOCATION))`

Reply:
@Tuubz Do you want to have 2 samples `rememberPermissionState` + `rememberMultiplePermissionsState ` or only `rememberMultiplePermissionsState`?